src/Microsoft.PowerShell.ConsoleHost/host/msh/ConsoleHostUserInterface.cs

```diff
@@ -33,7 +33,7 @@ internal partial class ConsoleHostUserInterface : System.Management.Automation.H
         /// <summary>
         /// This is the char that is echoed to the console when the input is masked. This not localizable
         /// </summary>
-        private const char printToken = '*';
+        private const char _printToken = '*';
```

Comment:
Maybe my diff is messed up, but I don't see where this is used with the new name in the two calls to 'ReadLineSave(false, printToken)...

Reply:
The diffs are a bit messed up since I updated quickly and then I ended up making a mistake. Since the commits are getting squashed at merged this should not affect the final history.

The final change is fixed in 338694dc2a14b5ffa6f24b9bdcd49aea942563e5